serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/MatrixClient.kt

```diff
@@ -462,15 +490,30 @@ class MatrixSession(val client: HttpClient, val server: String, val user: String
                             // Note that if this is the first event of this chunk, we record prev_batch for it
                             // so we can later backfill by picking the non-null prev_batch with smallest seqId
                             // for this room.
-                            println("adding event from sync at $insertId")
-                            Database.addRoomEvent(insertId, session_id, room_id, event, getRelatedEvent(event), if (index == 0 && room.timeline.limited) { room.timeline.prev_batch } else { null })
-                            insertId = idBetween(insertId, "z", true)
+                            if(event is RoomMessageEvent && event.redacts != null){
+                                println("this is a redaction")
```

Comment:
Remove this println, the later for loop has a loop where it prints out the latest redacted event.